main.css

```diff
@@ -1,49 +0,0 @@
-html {
```

Comment:
it appears that this file was *moved* without preserving the Git tracking...
```sh
mkdir css && git mv main.css ./css
```
see: http://git-scm.com/docs/git-mv